type-fest peer dependency warning

NPM warns that v0.4.3 requires a peer of type-fest@^0.13.1 when type-fest@^0.17.0 is installed.
Would it be possible to use **_"type-fest": ">=0.13.1"_** instead of **_"type-fest": "^0.13.1"_**?

this is s also breaking regular install on npm 7.0.4

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: type-fest@0.18.0
npm ERR! node_modules/type-fest
npm ERR!   dev type-fest@"0.18.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional type-fest@"^0.13.1" from @pmmmwh/react-refresh-webpack-plugin@0.4.3
npm ERR! node_modules/@pmmmwh/react-refresh-webpack-plugin
npm ERR!   dev @pmmmwh/react-refresh-webpack-plugin@"0.4.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!

--- a/package.json
+++ b/package.json
@@ -96,7 +96,7 @@
     "@types/webpack": "4.x",
     "react-refresh": ">=0.8.3 <0.10.0",
     "sockjs-client": "^1.4.0",
-    "type-fest": "^0.13.1",
+    "type-fest": ">=0.13.1 <1.0.0",
     "webpack": ">=4.43.0 <6.0.0",
     "webpack-dev-server": "3.x",
     "webpack-hot-middleware": "2.x",

